src/frontend/screens/WebView/index.tsx

```diff
@@ -34,7 +34,10 @@ import cn from 'classnames'
 
 function urlIsHpUrl(url: string) {
   const urlToTest = new URL(url)
-  return urlToTest.hostname === 'store.hyperplay.xyz'
+  return (
+    urlToTest.hostname ===
+    'hyperplay-store-git-feat-checkuptapi-hyperplay.vercel.app'
+  )
```

Comment:
I think we can do this to avoid hardcoding the URL twice
```ts
function urlIsHpUrl(url: string) {
  const urlToTest = new URL(url)
  return (
    urlToTest.hostname ===
    new URL(HYPERPLAY_STORE_URL).hostname
  )
}
```

Reply:
true